bitnami/thanos/templates/bucketweb/serviceaccount.yaml

```diff
@@ -1,9 +1,12 @@
-  
 {{- if .Values.bucketweb.enabled -}}
 apiVersion: v1
 kind: ServiceAccount
 metadata:
   name: {{ include "thanos.fullname" . }}-bucketweb
   labels: {{- include "thanos.labels" . | nindent 4 }}
     app.kubernetes.io/component: bucketweb
+  {{- if .Values.bucketweb.serviceAccount.annotations }}
```

Comment:
if `.Values.bucketweb.serviceAccount` doesn't exist it could fail, right? is it possible?

Reply:
Done.

Reply:
Yep.
For this case, I added empty value - .Values.bucketweb.serviceAccount.annotations: {}. because it also defined .Values.bucketweb.serviceAccount :)